site/docs-md/apis/keyboard/index.md

```diff
@@ -78,12 +78,12 @@ Keyboard.hide();
 
 ## Keyboard configuration (iOS only)
 
-Keyboard plugin allow this configurations for iOS
+Keyboard plugin allow this configurations for iOS. To be added to `capacitor.config.json`.
```

Comment:
"The keyboard plugin allows the following configuration values to be added in `capacitor.config.json` for the iOS platform:"